jobs/resource_job.go

```diff
@@ -74,6 +75,17 @@ type TaskDependency struct {
 	TaskKey string `json:"task_key,omitempty"`
 }
 
+// BEGIN Jobs + Repo integration preview
+type GitSource struct {
+	Url      string `json:"git_url" tf:"alias:url"`
+	Provider string `json:"git_provider,omitempty" tf:"alias:provider"`
```

Comment:
remove alias:provider, because `provider` is a reserved TF keyword...

Reply:
@edmondo1984 did you test creating real resource with locally built binary? here's how to do it https://github.com/databrickslabs/terraform-provider-databricks/blob/master/CONTRIBUTING.md#developing-provider

i just want to make sure `provider` attribute name is not clashing with a reserved tf keyword